examples/watch_pods.rs

```diff
@@ -71,7 +72,9 @@ async fn main() -> Result<()> {
     let mut prom = prometheus_client::registry::Registry::default();
 
     // Register application metrics before configuring the admin server.
-    let metrics = Metrics::register(prom.sub_registry_with_prefix("kubert_watch_pods"));
+    let metrics = Metrics::register(prom.sub_registry_with_prefix("pods"));
+    let runtime_metrics =
+        RuntimeMetrics::new(prom.sub_registry_with_prefix("kubert_resource_watch"));
```

Comment:
And then we'd configure a runtime with something like..
```suggestion
        RuntimeMetrics::new(prom.sub_registry_with_prefix("kubert"));
```